<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QWidget" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>653</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>653</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>重置密码</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;请输入你需要重置密码账号&lt;/p&gt;&lt;p&gt;绑定的邮箱或手机号&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="alineEdit_Account">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonGetCheck">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>获取验证码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入验证码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_CheckNum">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>验证新密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_NewPassword">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Check_NewPassword">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OK">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
